time_stamper_cv/src/led_detection_node.cpp

```diff
@@ -2,12 +2,13 @@
 #include "trigonometry.h"
 
 LedDetectionNode::LedDetectionNode() {
-  calibration_ = std::make_shared<ImageProcessor>(ImageProcessor(getConfiguration()));
+  image_processor_ = std::make_shared<ImageProcessor>(ImageProcessor(getConfiguration()));
 }
 
 void LedDetectionNode::init() {
-  img_pub_ = nh_.advertise<sensor_msgs::Image>("time_stamper_cv_image", 1);
-  calibration_->setVisualization(nh_private_.param("show_visualization", true));
+  img_pub_ = nh_.advertise<sensor_msgs::Image>("timestamper/image", 1);
+  led_state_pub_ = nh_.advertise<time_stamper_cv::Ledstate>("timestamper/led_state", 1);
```

Comment:
instead of namespacing the topic name, its probably better to just name them image and led_state and let the namespacing be done via roslaunch

Reply:
"Image" is ambiguous with topics from image_undistort, so I leave it like this